examples/falling-balls/README.md

```diff
@@ -16,7 +16,9 @@ Choose a run configuration for an appropriate target in Android Studio and run i
 
 `./gradlew desktopApp:run`
 
-## Run JS in browser with WebAssembly Skia via Gradle
+## Run JS in browser
```

Comment:
Discussed with @etolstoy. Decided that it is easier to remove the whole part about `Run JS in browser` in order not to explain what we don't support. If someone explores this target themselves, they can use it.

Reply:
Done. Remove run configuration in IDEA as well. Only code remains